org.eclipse.gef/.classpath

```diff
@@ -7,5 +7,6 @@
 	</classpathentry>
 	<classpathentry kind="con" path="org.eclipse.pde.core.requiredPlugins"/>
 	<classpathentry kind="src" path="src"/>
+	<classpathentry kind="src" path="icons"/>
```

Comment:
I'm quite sure this isn't needed.  None of the Platform bundles so this that:

https://github.com/eclipse-platform/eclipse.platform.ui/blob/master/bundles/org.eclipse.ui/.classpath

https://github.com/eclipse-platform/eclipse.platform.ui/blob/e3628a3a67c44771465d7d317b3424920c425090/bundles/org.eclipse.ui/plugin.xml#L1442-L1447

The build.properties will ensure the icons are in the final jar and it should work fine for debug launches as well.

Reply:
That value is indeed unnecessary. I've just removed it.